### PR TITLE
Fix test_group_version error for 1.2

### PR DESCRIPTION
**Pull request by Wenxing-hou**
Fix: #43 

And  test_group_version error has been fixed by this patch.

Signed-off-by: Wenxing Hou <wenxing.hou@intel.com>

### CODE REVIEW

**Comment by jyao1** [comment]
I think we can keep both values be valid.

**Comment by Wenxing-hou** [comment]
> I think we can keep both values be valid.

Yes. I have keep the both values.

File: doc/1.Version.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Assertion 1.2.3:
     SpdmMessage.SPDMVersion == 0x10
 
 Assertion 1.2.4:
-    SpdmMessage.Param1 == InvalidRequest.
+    (SpdmMessage.Param1 == InvalidRequest) OR (SpdmMessage.Param1 == VERSION_MISMATCH)
```
Comment (jyao1) [inline doc/1.Version.md:57]: We need use `VersionMismatch` in the spec.

Comment (Wenxing-hou) [reply doc/1.Version.md]: Thanks. I have changed it.

**Pull request merged**